framework/doc/content/source/materials/FVPropValPerSubdomainMaterial.md

```diff
@@ -0,0 +1,17 @@
+# FVPropValPerSubdomainMaterial
+
+!syntax description /Materials/FVPropValPerSubdomainMaterial
+
+## Overview
+
+This object is useful for providing a material property value, for finite volume
```

Comment:
I suppose that this is nice for FE too from a convenience standpoint; instead of having to specify multiple `GenericConstatMaterials` you can just create one object. But it's essential from a functional standpoint for finite volume calculations in order to circumvent the material property ghosting issue for one property name